src/TMxpNodeBuilder.cpp

```diff
@@ -35,6 +35,7 @@ TMxpNodeBuilder::TMxpNodeBuilder(bool ignoreText)
 , mIsInsideText(false)
 , mHasNode(false)
 , mIsText(false)
+, mIsEmptyTag(false)
```

Comment:
:information_source: This one must be repositioned between `mIsEndTag` and `mIsInsideTag` - as a rule the order here must match the order they are declared in the header file to prevent issues when early values are used in later initializations - as can happen when a class is derived from a base class... (which is the reason for the warning I mentioned you can get from Qt Creator in a previous point in this review)!